README.md

```diff
@@ -4,9 +4,199 @@ CartoDB Spatial Analysis extension for PostgreSQL.
 
 ## Code organization
 
-* *pg* contains the PostgreSQL extension source code
-* *python* Python module
+* *doc* documentation
+* *src* source code
+* - *src/pg* contains the PostgreSQL extension source code
+* - *src/py* Python module source code
+* *release* reseleased versions
+* *env* base directory for Python virtual environments
 
 ## Requirements
 
-* pip
+* pip, virtualenv, PostgreSQL
+* python-scipy system package (see src/py/README.md)
+
+# Working Process
+
+We use the default `develop` branch as the basis for development.
+This branch and `master` are maintained by the *Release Manager*.
+The `master` branch is used to merge and tag releases to be
+deployed in production.
+
+In addition to these two permanent branches, temporal *topic*
```

Comment:
I believe the correct adjective is "temporary"... anyway I'd rephrase this as: "A *topic branch* shall be created from `develop`  for any new feature or bugfix".